app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/settings/about/utils/CIBuildInfo.kt

```diff
@@ -0,0 +1,13 @@
+/**
+ * Contains build information provided by the CI build process.
+ *
+ * **WARNING:** This object is automatically generated by the CI scripts.
+ * Do not modify this file directly unless you are also updating the
+ * corresponding CI scripts that generate it.
+ */
+object CIBuildInfo {
+    /**
+     * Contains build information generated by the CI process.
+     */
+    val info: List<Pair<String, String>> = emptyList()
```

Comment:
I second that sentiment. Unless there's data we can't access from the gradle script we should be putting build tasks there instead of standalone scripts.

Reply:
Yeah that's the purpose of it, passing data from the workflow. iOS was done first and this followed a similar approach which helps maintaining them but, I don't oppose a different route as long as we're able to easily change the content without app code changes, having a map / keyvalue pairs list instead of individual variables.

Reply:
Even if we format the CI info in another location, there is more formatting taking place [here](https://github.com/bitwarden/android/pull/4456/files#diff-317dd91445d7e22f8ced693b2c1a3cd5dc6322fe7cd8ff2d28bb8dd1b878d1bcR126-R145), so it's likely app changes would be required regardless. IMO, it would be easier to maintain if all of the formatting for the copy was in one place. Some of the copy information is not available at compiletime, so formatting within the viewmodel at runtime seems like the most appropriate place.